src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,22 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+
+    @Override
+    public boolean test(Candidate candidate) {
+        int[] calculate = new int[2];
+        String [] years = candidate.getPeriodsInUkr().split("-");
+        for (int i = 0; i < years.length; i++) {
+            calculate[i] = Integer.parseInt(years[i]);
+        }
+        int peroid = calculate[1] - calculate[0];
```

Comment:
Here are just 2 indexes. No need to use loop here. It makes code difficult to read
```suggestion
        String [] years = candidate.getPeriodsInUkr().split("-");
        int firstYear = Integer.parseInt(years[0]);
        int lastYear = Integer.parseInt(years[1]);
        int peroid = lastYear - firstYear ;
```